docs/http.md

```diff
@@ -0,0 +1,61 @@
+# HTTP API
+
+When a Cog Docker image is run, it serves an HTTP API for making predictions.
+
+First, build your model:
+
+    cog build -t my-model
+
+Then, start the Docker container:
+
+    docker run -d -p 5000:5000 --init my-model
+
+    # If your model uses a GPU:
+    docker run -d -p 5000:5000 --init --gpus all my-model
+
+    # if you're on an M1 Mac:
+    docker run -d -p 5000:5000 --init --platform=linux/amd64 my-model
+
+Port 5000 is now serving the API:
+
+    curl http://localhost:5000
+
+To view the API documentation in browser for the model that is running, open [https://localhost:5000/docs](https://localhost:5000/docs).
```

Comment:
Does `https` work here?

Reply:
No, no, it does not...

Reply:
I just realized Copilot wrote this for me! Not my fault! Honest!